SickSangHae/ViewModel/CoreDataViewModel.swift

```diff
@@ -59,11 +106,75 @@ extension CoreDataViewModel {
         getAllReceiptData()
     }
     
-    func updateReceiptStateData(target: Receipt, state: Status) {
+    func updateStatus(target: Receipt, to status: Status) {
+        switch status {
+        case .shortTermUnEaten:
+            updateStatusToUnEaten(target: target, to: status)
+        case .shortTermPinned:
+            updateStatusToUnEaten(target: target, to: status)
+        case .longTermUnEaten:
+            updateStatusToUnEaten(target: target, to: status)
+        case .Eaten:
+            updateStatusToEaten(target: target)
+        case .Spoiled:
+            updateStatusToSpoiled(target: target)
```

Comment:
```suggestion
        case .shortTermUnEaten, .shortTermPinned, .longTermUnEaten:
            updateStatusToUnEaten(target: target, to: status)
        case .Eaten:
            updateStatusToEaten(target: target)
        case .Spoiled:
            updateStatusToSpoiled(target: target)
```

겹치는 케이스들은 함께 핸들링해도 될거가탸요 ㅎㅎ

Reply:
감사합니다 영
확실히 겹치는 케이스들을 합치는게 훨씬 깔끔해보이네요
코드가 계속 뚱뚱해져서 고민이었는데 아주 마음에 듭니다 감사합니다!